src/quo2/components/gradient/gradient_cover/view.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.gradient.gradient-cover.view
+  (:require [quo2.components.gradient.gradient-cover.style :as style]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.linear-gradient :as linear-gradient]))
+
+(defn- view-internal
+  [{:keys [color] :or {color :blue}}]
+  (let [color-top    (colors/custom-color-by-theme color 50 50 20 20)
+        color-bottom (colors/custom-color-by-theme color 50 50 0 0)]
+    [linear-gradient/linear-gradient
+     {:accessibility-label :gradient-cover
+      :colors              [color-top color-bottom]
+      :start               {:x 0 :y 0}
+      :end                 {:x 0 :y 1}
+      :style               style/root-container}]))
+
+(def view (quo.theme/with-theme view-internal))
```

Comment:
I'm not sure we actually need theme prop in this component but I think it might be best to keep this anyway so there is consistency across the components, hopefully we can fully migrate all of the components over soon :)

Reply:
Yes, @J-Son89, We don't require the `theme` as the colours are the same for both light and dark modes. I just wrapped it to keep it consistent with other components and later if the design team introduces separate colours for dark mode. This will be handy.